pages/learn/manage/device-statuses.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Device statuses
+excerpt: Determine the status and connectivity of a device
+---
+
+# Device statuses
+
+The application overview page shows the status of each device in the application.
+
+<img src="/img/common/main_dashboard/application_device_status.png" alt="Application device status" width="40%" >
+
+Each device can have one of the following statuses:
+
+| Status            | Description                                                                                                                                                                                                                   |
+|-------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Online            | The device is online and is connected to the VPN or has recent API communications.                                                                                                                                            |
+| Configuring       | The device is applying a configuration update. Additionally, this status is displayed during [host OS updates]host-os-updates, and for devices that require flashing, during the [provisioning][device-provisioning] process. |
```

Comment:
@garethtdavies I actually provided the extra context just so that you have the full picture to write the descriptions.
I wasn't requesting mentioning all the pre/post-provisioning process details. Sorry if I confused you.

For example here it could be as simple as
```suggestion
| Configuring       | The device is applying OS configuration. |
```

If you still think we should mention the details (which I think we don't need to be that verbose here), maybe first mention the provisioning part and the os update later, so that they are in a sequence order.

Reply:
Ah, perfect yes I did get confused by that and I agree it's overkill. I'll update it and thanks for providing that additional context 👍